website/src/css/_navbar.scss

```diff
@@ -22,3 +22,47 @@
   padding: 5px;
   font-weight: 300;
 }
+/*===========*/
+
+/* Fix navbar logo */
+.navbar__logo {
+  height: 26px;
+  margin: 0 10px 0 8px;
+}
+
+/* Restyle gamburger button */
+
+.navbar {
+  position: relative;
```

Comment:
Hmm, was it necessary to define `position: relative` here? I think it only complicated your task, because you had to apply an ugly override below:

```
div.searchBox_node_modules-\@docusaurus-theme-classic-lib-theme-Navbar-Search-styles-module {
   right: 80px;
 }
 ```
 
I think it's better if you remove that `position: relative`, so that your absolute positions are calculated against the outmost container (like it was before).

Reply:
Noticing that the height of the "navbar" block does not match the template, I determined it. And then I decided to move away from this block in determining indents, because then you can make corrections without taking into account the "padding" of the intermediate element. Come to think of it, it doesn't seem like such a good idea)